### ISSUE TITLE
More space on Dropbox !

### ISSUE THREAD

**Issue by yvantina** [creativedisturbance/administrative#62]
@porteriffic @coreysmart92 @kkondas Can't upload podcasts anymore. Need more space

**Comment by porteriffic** [comment]
@kkondas assigning this to you. Would like you to help manage the Dropbox in the future.

**Comment by kkondas** [comment]
@porteriffic I went through and backed-up and cleared podcast that I could tell have already been published. There should be room @yvantina to upload to dropbox.

**Comment by yvantina** [comment]
cool ! thanks ;)

**Comment by kkondas** [comment]
Welcome!

**Comment by cassininazir** [comment]
Almost there. Need to delete a bit more:

![screen shot 2015-06-02 at 3 33 27 pm](https://cloud.githubusercontent.com/assets/1661430/7946498/8fc7c516-093c-11e5-8c7d-685f364f317a.png)

**Comment by porteriffic** [comment]
@kkondas There is still not enough space on Dropbox. Did you clean up through your computer or through the Dropbox account?

**Comment by kkondas** [comment]
@porteriffic @cassininazir I did it from my account because I thought I might the ability to do it from there. I'm logged into Dropbox now through the C.D. log-in. I've checked all of the folders went through last night and the files are gone. When accessing the garbage can(hide deleted items) there are a couple of grayed out folders that I guess can be deleted one more time, but I'm not seeing any other files. Could this be from folders that C.D. is shared with?
